bootcamp/tutorials/integration/llama_agents_milvus.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>nit, the logo is a bit too large.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>